config/suppressions.xml

```diff
@@ -80,6 +80,7 @@
   <suppress checks="ClassFanOutComplexity" files="CheckstyleAntTask\.java"/>
   <suppress checks="ClassFanOutComplexity" files="CheckerTest\.java"/>
   <suppress checks="ClassFanOutComplexity" files="Checker\.java"/>
+  <suppress checks="ClassFanOutComplexity" files="XdocsPagesTest\.java"/>
```

Comment:
The annotation adds two new imports, because of which this test exceeds the limit.